src/style-stars--0.css

```diff
@@ -0,0 +1,3 @@
+.stars--0 .stars__star {
+  margin-top: 0;
+}
```

Comment:
It's not recommended to use tag names for styling. Instead of using `.stars__star` nested within `.stars--0`, assign a class to the elements you want to style and use that class for styling. This promotes reusability and avoids potential specificity issues.